.github/workflows/docker_upload.yml

```diff
@@ -0,0 +1,43 @@
+name: frr_exporter_docker_upload
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types: created
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
```

Comment:
Can you please hardcode this to avoid any future compatibility issues?

Reply:
Sure - I dislike doing this personally and just like to deal with problems as they arise, but each to their own.